### ISSUE TITLE
scuff-caspol broken?

### ISSUE THREAD

**Issue by acmcclung** [HomerReid/scuff-em#146]
Wasn't able to get the scuff-caspol tutorial example to work:

`scuff-caspol --PECPlate --atom Cesium --EPFile EPFile1
`

I've printed the EP and log files from running on my own machine, but the Simpetus AMI seems to produce the same error.

**EPFile1:**
`0.000000 0.000000 2.000`

**scuff-caspol.log:**
```
06/11/17::11:17:21: scuff-caspol running on Dionysus:30750 (06/11/17::11:17:21) 
/usr/local/bin/scuff-em/lib/libscuff.so.0(+0x7d6cd)[0x7f1343bf96cd]
/lib/x86_64-linux-gnu/libc.so.6(+0x360b0)[0x7f13432de0b0]
scuff-caspol[0x40214c]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed)[0x7f13432c97ed]
scuff-caspol[0x40273d]


**<><> received signal 11 
```

**Comment by HomerReid** [comment]
Fixed in c291e68. Thanks for the bug report.

Going forward I will probably merge scuff-caspol with scuff-ldos, since it doesn't really make sense to maintain two separate codes that do largely identical things. I would be curious to know what your use case is.

Feel free to ask more questions, or close the issue if satisfied.

**Comment by acmcclung** [comment]
Seems to be solved, thank you.

Our potential use case is pretty simple; we just want to know the CP potential for an atom next to a nanophotonic structure. I know there are [other methods](http://dx.doi.org/10.1088/1367-2630/15/8/083026) of calculating this sort of thing, but the simplicity of the setup in scuff-caspol is attractive. I will play with scuff-ldos; I assume [eq. (A.1)](http://dx.doi.org/10.1088/1367-2630/15/8/083026) can be adapted directly?
